ui/src/components/modals/DeleteModelVersionPyFuncV2Modal.js

```diff
@@ -0,0 +1,119 @@
+    { method: "DELETE", addToast: true, mock: mocks.noBody },
+    {},
+    false
+  );
+
+  const [jobs] = useMerlinApi(
+    `/models/${model.id}/versions/${version.id}/jobs`,
+    { mock: mocks.jobList },
+    []
+  );
+
+  const isActiveJob = function(status) {
+    return ["pending", "running"].includes(status)
+  }
+
+  useEffect(() => {
+    setActiveJob(jobs.data.filter(item => isActiveJob(item.status)))
+    setInactiveJob(jobs.data.filter(item => !isActiveJob(item.status)))
+  }, [jobs])
+
+  useEffect(() => {
+    if (isLoaded) {
+      closeModal();
+      callback();
+    }
+  }, [isLoaded, closeModal, callback]);
+
+  return (
+    <EuiOverlayMask>
+      <EuiConfirmModal
+        title="Delete Model Version"
+        onCancel={closeModal}
+        onConfirm={undeployVersion}
+        cancelButtonText="Cancel"
+        confirmButtonText="Delete"
+        buttonColor="danger"
+        confirmButtonDisabled={deleteConfirmation !== `${model.name}-version-${version.id}` || activeJob.length > 0}>
+        {isLoading && (
+          <EuiProgress
+            size="xs"
+            color="accent"
+            className="euiProgress-beforePre"
+          />
+        )}
+        {jobs.isLoading ? (
+                    <EuiProgress
+                    size="xs"
+                    color="accent"
+                    className="euiProgress-beforePre"
+                  />
+        ) : (
+          <div>
+            {activeJob.length > 0 ? (
+              <div>
+                You cannot delete this Model Version because there are <b> {activeJob.length} Active Prediction Jobs</b> using this version. 
+                <br/> <br/> If you still wish to delete this model version, please <b>Terminate</b> Prediction Jobs that use this version or wait until the job completes.         
+              </div>
+              
+            ) : (
+              <div>
+                You are about to delete model <b>{model.name}</b> version <b>{version.id}</b>. This action cannot be undone. 
+                
+                <br/> <br/> To confirm, please type "<b>{model.name}-version-{version.id}</b>" in the box below
+                  <EuiFieldText     
+                    fullWidth            
+                    placeholder={`${model.name}-version-${version.id}`}
+                    value={deleteConfirmation}
+                    onChange={(e) => setDeleteConfirmation(e.target.value)}
+                    isInvalid={deleteConfirmation !== `${model.name}-version-${version.id}`} />  
+              </div>
+            )}
+            <br></br>
+            {activeJob.length === 0 && inactiveJob.length > 0 && (
+                <span>Deleting this Model Version will also delete {inactiveJob.length} <b>Inactive</b> Prediction Jobs using this version. <br/> <br/></span>
+            )}
+            <JobsTableModelVersion jobs={activeJob.length > 0 ? activeJob : inactiveJob} isLoaded={jobs.isLoaded} error={jobs.error} />
```

Comment:
Same comment as `ui/src/components/modals/DeleteModelVersionModal.js` on the placement of this `<JobsTableModelVersion />` block.